### ISSUE TITLE
Cannot bundle functions with Chinese character names

### ISSUE THREAD

**Issue by sgalal** [purescript/purescript#3804]
(Previously reported in https://github.com/purescript/spago/issues/590)

Steps to reproduce:

```sh
$ mkdir TestProject
$ cd TestProject
$ spago init
```

Modify `/src/Main.purs`:

```purescript
module Main where

import Prelude

import Effect (Effect)
import Effect.Console (log)

_試 :: Effect Unit
_試 = do
  log "🍝"

main :: Effect Unit
main = _試
```

Then:

```sh
$ spago run
[info] Installation complete.
Compiling Main
[info] Build succeeded.
🍝
```

```sh
$ spago bundle-app --main Main --to index.js
[info] Installation complete.
[info] Build succeeded.
Error in module Main (Regular):

  The module could not be parsed:
  lexical error @ line 4 and column 6
[error] Bundle failed.
```

The command invoked by spago is:

```sh
$ purs bundle "output/*/*.js" -m Main --main Main -o index.js
Error in module Main (Regular):

  The module could not be parsed:
  lexical error @ line 4 and column 6
```

**Comment by hdgarrood** [comment]
Thanks for the report! This looks like a potential bug in language-javascript to me, since it is failing on this line:
```javascript
var _試 = Effect_Console.log("\ud83c\udf5d");
```
but _試 is valid as a JavaScript identifier.

**Comment by JordanMartinez** [comment]
Looks like this is also resolved via #4255
